tests/unit/amo/pages/TestLandingPage.js

```diff
@@ -622,4 +608,16 @@ describe(__filename, () => {
       partialContent,
     );
   });
+
+  it.each([ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME])(
```

Comment:
Testing all values seems unnecessary because we already know that `getVisibleAddonType()` handles all values. Thus, you can test one value and trust that it will work for all values.